guides/framework-integration/react.md

```diff
@@ -93,5 +93,17 @@ import './app.jsx';
 {% endtab %}
 {% endtabs %}
 
+{% hint style="info" %} Note with React 18, the file `src/app.jsx` must be updated as follows:
```

Comment:
This differs from the approach in https://github.com/electron-forge/electron-forge-docs/pull/93

We should choose the better (?) one.